src/rgw/rgw_op.cc

```diff
@@ -1945,20 +1945,20 @@ void RGWGetUsage::execute()
 
   op_ret = rgw_user_sync_all_stats(store, s->user->user_id);
   if (op_ret < 0) {
-    ldout(store->ctx(), 0) << "ERROR: failed to sync user stats: " << dendl;
+    ldout(store->ctx(), 0) << "ERROR: failed to sync user stats" << dendl;
     return;
   }
 
   op_ret = rgw_user_get_all_buckets_stats(store, s->user->user_id, buckets_usage);
   if (op_ret < 0) {
-    cerr << "ERROR: failed to sync user stats: " << std::endl;
+    cerr << "ERROR: failed to get user's buckets stats" << std::endl;
```

Comment:
oops, could you change this from `cerr` to `ldout` while you're at it?

Reply:
@cbodley done！